### PR TITLE
 Fix python 3.6+ compatibility when iterating over babel dicts

### PR DESCRIPTION
**Pull request by amatissart**
Hi!
On python 3.6+, day names are offset by one.  
Latest versions of python preserve insertion order in dicts, and Sunday comes first in Babel data for some reason. So it is necessary to sort them by key explicitly.

Why not setting up Travis and run the tests on multiple versions of Python automatically ?

### CODE REVIEW

**Comment by amatissart** [comment]
And that should fix #20 :)

File: humanized_opening_hours/main.py [hunk 1]
```diff
@@ -628,13 +628,13 @@ def get_localized_names(self):
...
+        month_names_items = babel.dates.get_month_names(locale=self.locale).items()
         return {
-            "days": list(
-                babel.dates.get_day_names(locale=self.locale).values()
-            ),
-            "months": list(
-                babel.dates.get_month_names(locale=self.locale).values()
-            )
+            # names sorted by day index (from 0:Monday to 6:Sunday)
+            "days": list(day_name for _, day_name in sorted(day_names_items)),
```
Comment (rezemika) [inline humanized_opening_hours/main.py]: It's quite superficial, but why using `list(day_name for _, day_name in sorted(day_names_items))` and not `[day_name for _, day_name in sorted(day_names_items)]`?

File: requirements-dev.txt [hunk 2]
```diff
@@ -4,7 +4,7 @@ certifi==2018.1.18
 chardet==3.0.4
 flake8==3.5.0
 idna==2.6
-lark-parser==0.5.3
+lark-parser==0.6.4
```
Comment (rezemika) [inline requirements-dev.txt]: It's totally relevent. However, I think it isn't related to this PR in particular, isn't?

File: humanized_opening_hours/main.py (hunk 1 above)
Comment (rezemika) [reply humanized_opening_hours/main.py]: Thank you so much for this fix!
Sorry for the late, I was so busy the last days.

File: requirements-dev.txt (hunk 2 above)
Comment (amatissart) [reply requirements-dev.txt]: It's not related. But some tests fail when using lark-parser==0.5.3. So I suppose 0.6 is now required ?

Comment (rezemika) [reply requirements-dev.txt]: Oh sorry, you're right. The 0.6.0 of Lark broke backward compatibility on some points, I forgot this point.

File: humanized_opening_hours/main.py (hunk 1 above)
Comment (amatissart) [reply humanized_opening_hours/main.py]: No problem! Using list comprehension is more pythonic indeed, I updated the PR.

Travis is probably quite easy to set up, but I can understand that's not a priority.

**Comment by amatissart** [comment]
Thanks!

**Pull request closed without merging**